src/commands/org/open.ts

```diff
@@ -130,48 +123,21 @@ export class OrgOpenCommand extends SfCommand<OrgOpenOutput> {
       throw err;
     }
 
-    const openOptions = flags.browser
-      ? // assertion can be removed once oclif option flag typings are fixed
-        { app: { name: apps[flags.browser as 'chrome' | 'edge' | 'firefox'] } }
-      : {};
+    const openOptions =
+      // assertions could be removed once oclif flag typins are fixed
+      flags.browser ? { app: { name: apps[flags.browser as 'chrome' | 'edge' | 'firefox'] } } : {};
 
-    await utils.openUrl(url, openOptions);
-    return output;
-  }
+    // create a local html file that contains the POST stuff.
+    // for review...there's always an access token, right?
+    const tempFilePath = path.join(tmpdir(), `org:open-${new Date().valueOf()}.html`);
```

Comment:
Replace colons in filename with hyphen or underscore, colons aren't allowed in file paths in Windows
https://stackoverflow.com/questions/1976007/what-characters-are-forbidden-in-windows-and-linux-directory-names